bin/test.py

```diff
@@ -204,14 +213,16 @@ def wait_for_stacks(stacks, create):
 def ensure_foundation(session, config):
     """ Make sure we have everything we need in place to run the stacks
     """
+    key_pairs=[]
     for region in get_regions(session.client('ec2', region_name='us-east-1', config=config)):
 
         print 'Ensure key pair in region: {}'.format(region)
 
         ec2_client = session.client('ec2', region_name=region, config=config)
         key_name = '{}{}'.format(KEY_PAIR_PREFIX, region)
         if not has_key_pair(ec2_client, key_name):
-            store_key_locally(key_name, create_key_pair(ec2_client, key_name))
```

Comment:
This function is not in use and should be removed.

Reply:
Removed function.